### PR TITLE
Change default render relay plane to game relay

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Game relay is a more sane default than the master plate

## Changelog

:cl:
code: changed a code-related default, please report if anything turns invisible
/:cl:

### CODE REVIEW

File: code/_onclick/hud/rendering/plane_master.dm [hunk 1]
```diff
@@ -266,3 +250,4 @@
 	plane = FIELD_OF_VISION_BLOCKER_PLANE
 	render_target = FIELD_OF_VISION_BLOCKER_RENDER_TARGET
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	render_relay_plane = null
```
Comment (Timberpoes) [inline code/_onclick/hud/rendering/plane_master.dm:253]: The render_relay_plane used to be RENDER_PLANE_MASTER by inheritance. What does setting this to null change?

File: code/_onclick/hud/rendering/plane_master.dm [hunk 2]
```diff
@@ -82,7 +77,6 @@
 /atom/movable/screen/plane_master/game_world_fov_hidden
 	name = "game world fov hidden plane master"
 	plane = GAME_PLANE_FOV_HIDDEN
-	render_relay_plane = GAME_PLANE
```
Comment (Timberpoes) [inline code/_onclick/hud/rendering/plane_master.dm:85]: GAME_PLANE is defined as -4, setting this to inherit the value of RENDER_PLANE_GAME changes it to 990.

Is this a fix and what does this change?

In addition, `/atom/movable/screen/plane_master/game_world_above` also has `render_relay_plane = GAME_PLANE` but is unchanged?

**Review by Timberpoes — COMMENTED** [review]
Looking at the changelog brings up inconsistencies.

File: code/_onclick/hud/rendering/plane_master.dm (hunk 2 above)
Comment (TiviPlus) [reply code/_onclick/hud/rendering/plane_master.dm]: typo

File: code/_onclick/hud/rendering/plane_master.dm (hunk 1 above)
Comment (TiviPlus) [reply code/_onclick/hud/rendering/plane_master.dm:254]: It does not relay

**Pull request merged**